### PR TITLE
feat(i18n): add `zh-TW` translation

### PR DESCRIPTION
**Pull request by benben6515**
add some `zh-TW` first, when I have more time I will be back!

---
- 2023/02/05 updated

- [x] Translated `zh-TW` 100% for now :D

If the core team wants to change some of them for good, go for it. Maybe I just add rough translations.

### CODE REVIEW

File: locales/zh-TW.json [hunk 1]
```diff
@@ -112,7 +122,13 @@
     },
     "common": {
       "cancel": "否",
-      "confirm": "是"
+      "confirm": "是",
+      "title": "你確定嗎？"
```
Comment (Mini-ghost) [inline locales/zh-TW.json:126]: This doesn't seem to be used in Elk.

File: locales/zh-TW.json [hunk 2]
```diff
@@ -138,33 +154,69 @@
...
+      "lines": "第 {0} 行",
+      "open": "開啟",
+      "snippet_from": "取自 {0} 的片段"
+    }
+  },
   "error": {
     "account_not_found": "未找到使用者 {0}",
     "explore-list-empty": "目前沒有熱門話題，稍後再來看看吧！",
-    "file_size_cannot_exceed_n_mb": "檔案大小不能超過 {0}MB",
+    "file_size_cannot_exceed_n_mb": "檔案大小不能超過 {0} MB",
```
Comment (Mini-ghost) [inline locales/zh-TW.json:167]: "檔案大小不能超過 10 MB" seems a bit odd ?

File: locales/zh-TW.json [hunk 3]
```diff
@@ -138,33 +154,69 @@
...
     "block_domain": "封鎖的域名 {0}",
     "copy_link_to_post": "複製這篇貼文的連結",
+    "copy_original_link_to_post": "複製貼文的原網址",
     "delete": "刪除",
     "delete_and_redraft": "刪除並重新編輯",
+    "delete_confirm": {
+      "cancel": "取消",
+      "confirm": "確認",
+      "title": "你確定要刪除這篇貼文嗎？"
+    },
```
Comment (Mini-ghost) [inline locales/zh-TW.json:219]: This doesn't seem to be used in Elk.

File: locales/zh-TW.json [hunk 4]
```diff
@@ -193,16 +245,19 @@
...
     "explore": "探索",
     "favourites": "喜歡",
     "federated": "聯邦",
     "home": "首頁",
+    "list": "列表",
+    "lists": "列表",
     "local": "本站",
     "muted_users": "已靜音的使用者",
     "notifications": "通知",
+    "privacy": "隱私條款",
```
Comment (Mini-ghost) [inline locales/zh-TW.json:260]:
```suggestion
    "privacy": "隱私權政策",
```

File: locales/zh-TW.json [hunk 5]
```diff
@@ -264,32 +321,54 @@
...
     "account_settings": {
       "description": "在 Mastodon UI 中編輯你的帳號設定",
       "label": "帳號設定"
     },
+    "feature_flags": {
+      "github_cards": "GitHub 卡片",
+      "title": "實驗性功能",
+      "user_picker": "使用者設置",
+      "virtual_scroll": "虛擬滾動"
+    },
```
Comment (Mini-ghost) [inline locales/zh-TW.json:344]: This doesn't seem to be used in Elk.

File: locales/zh-TW.json [hunk 6]
```diff
@@ -264,32 +321,54 @@
...
       "font_size": "字體大小",
       "label": "外觀",
       "light_mode": "淺色",
+      "size_label": {
+        "lg": "大",
+        "md": "中",
+        "sm": "小",
+        "xl": "特大",
+        "xs": "特小"
+      },
```
Comment (Mini-ghost) [inline locales/zh-TW.json:358]: This doesn't seem to be used in Elk.

File: locales/zh-TW.json [hunk 7]
```diff
@@ -318,10 +397,13 @@
         "save_settings": "儲存設定變更",
         "subscription_error": {
           "clear_error": "清除錯誤",
+          "invalid_vapid_key": "VAPID 公鑰似乎無效。",
           "permission_denied": "權限不足：請在你的瀏覽器中打開通知權限。",
+          "repo_link": "鹿鳴的 GitHub 倉庫",
```
Comment (Mini-ghost) [inline locales/zh-TW.json:402]: How about use 儲存庫 instead of 倉庫 ?

File: locales/zh-TW.json (hunk 2 above)
Comment (benben6515) [reply locales/zh-TW.json:167]: Maybe it is my preference, but I think "檔案大小不能超過 10 MB" are more readable.

REF: 'https://support.microsoft.com/zh-tw/office/使用-outlook-傳送大型檔案-8c698842-b462-4a4c-8d53-5c5dd04f77ef'

Again, choose what you want!

File: locales/zh-TW.json (hunk 3 above)
Comment (benben6515) [reply locales/zh-TW.json:219]: Then, should I remove these XD

File: locales/zh-TW.json (hunk 7 above)
Comment (benben6515) [reply locales/zh-TW.json:402]: That makes sense!

File: locales/zh-TW.json (hunk 3 above)
Comment (Mini-ghost) [reply locales/zh-TW.json:219]: I think it's better to remove it.

Comment (benben6515) [reply locales/zh-TW.json]: Of course, then I remove your mentioned parts.

**Review by Mini-ghost — APPROVED** [review]
LGTM

**Comment by xatier** [comment]
@benben6515 FYI, related PR https://github.com/elk-zone/elk/pull/1617

**Comment by benben6515** [comment]
@xatier Thank you for sharing this information. Honestly, '您' and '你' both work to me.
Therefore, I think we just provide references and let `elk` team decide which one. 🫡

**Comment by xatier** [comment]
@benben6515 exactly, the existing `zh-tw` translation is using a mixed tone, which we should make it really consistent. Once the elk team picks a choice, let's update this PR.

**Pull request merged**